src/Inventory/Asset/InventoryNetworkPort.php

```diff
@@ -476,8 +476,11 @@ private function handleUpdates()
                             if ($ip_data == $db_ip_data) {
                                 unset($ips[$ip_key]);
                                 unset($db_addresses[$db_ip_key]);
-                             //result found in db, useless to continue
-                                break 1;
+                                //result found in db, useless to continue
```

Comment:
Should we keep the comment if you remove the `break` line ?

Reply:
done